index.md

```diff
@@ -19,11 +19,9 @@ Specifically, our supporters have provided:
 
 **This directory is open-source.** Any firm can add and update their policies and point(s)-of-contact, and we hope that every major venture firm will participate.
 
-<div align="center">
-<button>
-<a href="https://airtable.com/shrudsWooa1wa7sCm">Join #MovingForward</a>
-</button>
-</div>
+<p style="text-align:center">
```

Comment:
Preferable to not inline styles in HTML - can you put a classname on the `<p>` and style it from CSS?

Reply:
yea sure, was just going off what was there before but happy to change it one sec